docs/launch-arbitrum-chain/02-configure-your-chain/common-configurations/04-stake-and-validator-configurations.mdx

```diff
@@ -1,12 +1,137 @@
 ---
 title: 'Stake and validator configurations'
 description: 'Learn how to configure your Arbitrum chain with a custom stake and validator configurations'
-author:
+author: pete-vielhaber
 sme:
 content_type: how-to
 tags: ['hide-from-search']
 ---
 
-- **Base Stake**: Minimum bond required for validators.
-- **Stake Token**: Token used for staking.
-- **Loser Stake Escrow**: Address for escrowed funds from losing validators.
+Arbitrum Orbit chains are customizable Layer 3 (L3) chains that settle to an Arbitrum Layer 2 (L2) chain, such as Arbitrum One. They support validator configurations to ensure chain security through bonding (staking) and assertion challenges. Validators post assertions about the chain's state on the parent L2 chain and can challenge incorrect assertions. Orbit chains can be permissioned, meaning validators must be whitelisted.
+
+Staking is required for active validation, where validators bond funds to participate. If a validator loses a challenge (e.g., due to a faulty assertion), their bonded stake is escrowed or burned. Configurations such as stake token, base stake, and loser stake escrow are configurable during chain deployment or post-deployment via contract calls.
```

Comment:
Some of those definitions could probably use quicklooks...Or modal hover overs :)